apps/windows/package.json

```diff
@@ -48,7 +42,14 @@
     "ts-node": "^8.10.1",
     "tsconfig-paths": "^3.9.0",
     "typescript": "3.8.3",
-    "webdriverio": "5.22.4"
+    "webdriverio": "^6.5.0",
+    "appium-mac2-driver": "0.14.1",
```

Comment:
same comment about windows

Reply:
No, good catch. Removing for Win32/Windows now. Thanks